src/index.html

```diff
@@ -6,7 +6,43 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar calendar--start-day-sun calendar--month-length-28">
```

Comment:
```suggestion
      <div class="calendar calendar--start-day--sun calendar--month-length--28">
```
<img width="849" alt="image" src="https://user-images.githubusercontent.com/102411130/235079029-449eaf37-be9f-45b8-8957-8a52c7e87c0c.png">